src/app/news/news-list-table-rows/news-list-table-rows.component.html

```diff
@@ -1,7 +1,7 @@
 <tr
   *ngFor="let activity of activities | paginate: { id: 'table-template-pagination', itemsPerPage: paginationData.pageSize, currentPage: paginationData.currentPage, totalItems: paginationData.totalListItems }">
   <td scope="row" data-label="Headline" class="col-10">
-    <span class="font-weight-bold">{{activity.project.name || '-'}}</span>
```

Comment:
We want to preserve this functionality. ie. if there is no project name then put a - instead